docs/docs/visual-testing-with-storybook.md

```diff
@@ -11,20 +11,14 @@ As your project grows over time having this information available will be invalu
 
 ## Setting up your environment
 
-To set up Storybook you need to install dependencies and do some custom configuration. First, install the Storybook CLI.
-
-```sh
-npm install -g @storybook/cli
-```
-
-Once the CLI is installed, the next step is to run the `getstorybook` command that is now available from the root directory of your Gatsby project.
+To set up Storybook you need to run `sb init`, which is Storybook's CLI tool. It's recommended to run it using `npx` so that you're always using the latest version:
 
 ```sh
 cd my-awesome-gatsby-project
-getstorybook
+npx -p @storybook/cli sb init
```

Comment:
afaik `npx` is not supported (out of the box on node@6 which we still support)

Reply:
It _might_ be OK to document using `npx` with more "advanced" tutorials like this. That said, I like being consistent here, too, so yeah, we should probably stick to globally installed executables.

Reply:
I'll let you guys make the call on this. It looks great from a docs perspective!

Reply:
Ok, so npm 5.2.0 (which includes Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby supports Node 6 and up, documentation should only introduce `npx` as an optional command in a note like so:

> npm 5.2.0 (which includes Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby supports Node 6 and up, we introduce `npx` here as an optional command for users of npm 5.2.0.